tests/unit/IonTextReaderTest.js

```diff
@@ -125,6 +125,29 @@
       assert.equal(ionReader.fieldName(), "innerkey2");
     };
 
+      suite['Accept IVM like symbols and throw on IVMs except $ion_1_0'] = function() {
+          let shouldPass = ["$ion_1_0", "$ion_schema_1_0", "$ion_1", "$ion_1_a", "$ion_", "ion_1_"];
```

Comment:
shouldPass/shouldFail doesn't tell the whole story, does it?  shouldFail is really shouldThrow, and some of the values in shouldPass should produce `true`, while others should produce `false`.  let's assert the expected behavior for all test values

i'm thinking of something more along the following lines:
```javascript
suite['...'] = function() {
  test_isIVM("$ion_1_0", true);
  test_isIVM("$ion_schema_1_0", false);
  test_isIVM("$ion_2_0", "throw");
  ...
}

function test_isIVM(value, expect) {
  actual = isIVM(value);
  if (expect == "throw") {
    throw new Error("Expected " + value + " to throw an error.");
  }
  assert expect == actual;
}
```

thoughts?

Reply:
I think the description of the suite does the same thing?

Reply:
not sure what you mean.  the description is just a string--it doesn't do anything to verify that `isIVM()` either returns the expected value or throws

Reply:
same response as above! I'll get right on it.